addons/website/static/tests/tours/page_manager.js

```diff
@@ -11,11 +11,12 @@ const checkKanbanGroupBy = [{
     extra_trigger: '.o_kanban_renderer',
     trigger: '.o_control_panel .o_searchview_dropdown_toggler',
 }, {
-    content: "Click on Add Custom Group",
-    trigger: '.o_add_custom_group_menu',
-}, {
-    content: "Click on Apply", // Active is selected by default
-    trigger: '.o_add_custom_group_menu + .o_accordion_values .btn-primary',
+    content: "Select 'Active' in the select of Add Custom Group",
+    trigger: ".o_add_custom_group_menu",
```

Comment:
I think 
`text: "active",`
could work here.
In any case there is a simpler way to work with select elements